docs/en/ingest-management/troubleshooting/troubleshooting.asciidoc

```diff
@@ -402,6 +402,10 @@ Note that the diagnostics bundle is intended for debugging purposes only, its st
 IMPORTANT: {agent} attempts to automatically redact credentials and API keys when creating diagnostics. 
 Please review the contents of the archive before sharing to ensure that there are no credentials in plain text.
 
+IMPORTANT: When enabling `debug` logging level, the zip archive containing diagnostics information will also contain
+the raw events of documents sent to the output. Please review the contents of the archive before sharing to ensure
+that there is no sensible information in them.
+
```

Comment:
```suggestion
IMPORTANT: The zip archive containing diagnostics information will contain the raw events of documents sent to the output. By default, it will log only the failing events as `warn`.
When enabling `debug` logging level, it will log all the events. 
Please review the contents of the archive before sharing to ensure that there is no sensible information in them.
```